AWSSDKSwiftCLI/Sources/AWSCLIUtils/Version.swift

```diff
@@ -7,15 +7,46 @@
 
 import Foundation
 import ArgumentParser
-import PackageDescription
 
-// MARK: - Decodable
```

Comment:
The struct below replaces the one that was previously imported from the `SwiftPM` package.